concepticondata/conceptlists/Goz-2017-600.tsv

```diff
@@ -0,0 +1,601 @@
+Goz-2017-600-549	549	uyum	harmony			3.34	2.27	3.29	4.17	3.6	2.74	2.06	139	2	4
+Goz-2017-600-550	550	uyumlu	harmonious			3.34	2.32	3.12	3.78	4	2.89	2.1	55	3	6
+Goz-2017-600-551	551	uzak	remote			1.81	1.61	1.75	1.95	1.92	3.65	2.87	444	2	4
+Goz-2017-600-552	552	uzay	space	3006	SPACE (AVAILABLE)	3.7	2.53	3.35	4.05	4.73	4.89	4.64	62	2	4
+Goz-2017-600-553	553	uzun	long	1203	LONG	1.84	1.57	1.79	2.2	1.8	4.65	4.68	781	2	4
+Goz-2017-600-554	554	ücret	fee			3	2.04	2.82	3.6	3.46	4.39	5.2	177	2	5
+Goz-2017-600-555	555	ülser	ulcer	1189	ULCER	5.37	4.06	5.75	5.34	5.8	3.44	5.1	16	2	5
+Goz-2017-600-556	556	ümitli	hopeful			3.26	2.23	3.57	3.9	3.35	2.8	2.1	3	3	6
+Goz-2017-600-557	557	üretici	producer			3.49	2.34	3.81	4.25	3.57	3.35	5.06	100	4	7
+Goz-2017-600-558	558	üşengeç	procrastinating			3.37	2.58	3.7	4.06	3.15	3.62	3.06	1	3	7
+Goz-2017-600-559	559	ütopya	utopia			5.83	3.43	6.21	6.37	6.47	3.04	1.61	11	3	6
+Goz-2017-600-560	560	üzüm	grapes	920	GRAPE	1.52	1.47	1.4	1.64	1.55	6.57	6.93	61	2	4
+Goz-2017-600-561	561	vali	governor			3.39	2.75	3.65	3.81	3.36	5.09	6.11	80	2	4
+Goz-2017-600-562	562	varlık	existence			2.95	2.1	3.48	3.4	2.89	3.14	3.78	349	2	6
+Goz-2017-600-563	563	vatan	homeland			2.52	2.19	2.79	2.89	2.28	4.06	4.43	58	2	5
+Goz-2017-600-564	564	vicdan	conscience			3.47	2.46	3.62	4.03	3.75	2.32	1.39	75	2	6
+Goz-2017-600-565	565	virüs	virus			4.61	3.28	4.44	5.28	5.35	3.13	5.13	97	2	5
+Goz-2017-600-566	566	viski	whisky			4.95	3.11	4.76	5.61	6.04	6.05	6.83	27	2	5
+Goz-2017-600-567	567	vitrin	showcase			3.43	2.46	3.34	3.89	3.96	5.99	6.56	64	2	6
+Goz-2017-600-568	568	yağcılık	flattery			4.2	2.86	4.3	4.78	4.81	3.07	2.39	4	3	8
+Goz-2017-600-569	569	yağış	precipitation			2.32	2.01	2.73	2.57	2.03	5.87	6.04	24	2	5
+Goz-2017-600-570	570	yağmur	rain	658	RAIN (PRECIPITATION)	1.64	1.58	1.63	1.71	1.64	6.45	6.55	160	2	6
+Goz-2017-600-571	571	yakıt	fuel			3.21	2.43	3.45	3.58	3.39	4.98	6.48	72	2	5
+Goz-2017-600-572	572	yalan	lie	2826	LIE (FALSEHOOD)	2.05	1.84	2.02	2.29	2.04	2.68	2.35	127	2	5
+Goz-2017-600-573	573	yalı	seaside			3.67	2.59	3.28	4.3	4.34	6.02	6.73	18	2	4
+Goz-2017-600-574	574	yama	patch			2.76	2.44	3.89	2.85	2.06	5.41	6.45	1	2	4
+Goz-2017-600-575	575	yangın	fire	221	FIRE	1.95	1.71	2.09	2.17	1.85	5.74	6.07	103	2	6
+Goz-2017-600-576	576	yansız	impartial			4.24	2.8	4.5	4.86	4.39	2.59	2.29	6	2	6
+Goz-2017-600-577	577	yara	wound	1109	WOUND	1.71	1.68	1.69	1.81	1.66	5.71	6.21	82	2	4
+Goz-2017-600-578	578	yardım	help	3467	HELP (ASSISTANCE)	2.12	1.87	2.23	2.4	2.02	3.47	2.92	289	2	6
+Goz-2017-600-579	579	yarım	half	816	HALF	2.06	1.72	2.02	2.43	2.06	4.52	4.37	131	2	5
+Goz-2017-600-580	580	yarış	race			2.02	1.7	1.99	2.32	2.06	4.81	4.83	101	2	5
+Goz-2017-600-581	581	yastık	pillow	1044	PILLOW	1.44	1.38	1.41	1.5	1.45	6.57	6.91	77	2	6
+Goz-2017-600-582	582	yatsı	nighttime			2.8	2.33	3.54	3.1	2.36	3.6	4.39	1	2	5
+Goz-2017-600-583	583	yavru	offspring			1.82	1.61	1.76	2.13	1.8	4.85	5.52	122	2	5
```

Comment:
Judging by the Google images (;D), the one above refers to human offspring, while this one - to animals AND humans. I also checked here: https://educalingo.com/en/dic-tr/evlat

Reply:
I see!